File: source/frontend/src/app/entities/vacature/postbusadresBuitenland.ts
import {PostbusadresBuitenlandImpl} from './postbusadresBuitenlandImpl';

export interface PostbusadresBuitenland {
  postbusadresbuitenland?: PostbusadresBuitenlandImpl;
}

export function isPostbusadresBuitenland(object: any): object is PostbusadresBuitenland {
  return 'postbusadresbuitenland' in object;
}

File: source/backend-vacatures-bemiddelaar/src/main/java/com/cap/pocvng/entity/PostbusadresBuitenland.java
package com.cap.pocvng.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.validation.Valid;

/**
 * PostbusadresBuitenland
 */


@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class PostbusadresBuitenland implements AdresDetailsBuitenland {

    @Valid
    private PostbusadresBuitenlandImpl postbusadresbuitenland;

}
